definitions/variable/emissions/tag_land-removal-options.yaml

```diff
@@ -0,0 +1,16 @@
+- Land Removal Option:
+    - Agroforestry:
+        description: intentionally integrating trees with crops or pasture on the same land
+    - Re/Afforestation:
```

Comment:
Still think this needs a subcategory. I tried to commit a PR, but it didn't work through the web interface:

```
    - Re/Afforestation:
        description: regrowing forested area on previously forested area (natural forest) or growing forested area on previously non-forested area (monoculture/non-native species)
    - Re/Afforestation|Reforestation:
        description: regrowing forested area on previously forested area (natural forest)
    - Re/Afforestation|Afforestation:
        description: growing forested area on previously non-forested area (monoculture/non-native species)
```

Reply:
Separate question - how does a `\` work? Would we want to use something else (e.g. `-`)...

Reply:
Well, @flohump initially suggested "Re-Afforestation", but I vetoed this because it can be very confusing (like reforesting only on previously afforested areas?) - see https://github.com/IAMconsortium/common-definitions/pull/180#issuecomment-2443213638 and subsequent preference for `/`.

About the additional sub-categories, I suggest to merge this PR as is, wait for the parallel PR on AFOLU variables to be merged, and then implement a parallel solution in he carbon-removal variable tree.

Reply:
@flohump do you agree on this sub-categorization? Can we expect a parallel PR as @danielhuppmann suggests?

Reply:
I think it would be more meaningful to split the sub-categories based on the type of Re/Afforestation (natural forest vs. plantation), which is more informative in particular for biodiversity. 
We will have the same differentiation in the land cover variables. 
My suggest would be: 
Re/Afforestation:
        description: Reforestation and/or Afforestation for carbon sequestration
Re/Afforestation|Natural Forest:
        description: Reforestation and/or Afforestation for carbon sequestration with native tree species 
      resembling natural vegetation
Re/Afforestation|Plantation:
        description: Reforestation and/or Afforestation for carbon sequestration with non-native 
      species and/or as monoculture plantation